scss/civicrm/administer/settings/_smtp.scss

```diff
@@ -2,43 +2,29 @@
   .crm-form-block {
     @include civicrm-table;
 
-    background: none;
-    box-shadow: none;
-
     table.form-layout-compressed {
-      @include civicrm-panel;
 
-      width: 100%;
+      td {
+        input[type=radio] + label {
+          line-height: 135%;
```

Comment:
again, this percentage value looks very weird

Reply:
Removed based on new changes of radio/checkboxes.